spec/lib/task-graph-spec.js

```diff
@@ -520,6 +520,14 @@ describe('Task Graph', function () {
                     .that.equals('finished');
             });
         });
+
+        it('should create task with waitOn.either', function(){
```

Comment:
@zhangy42 : I would use the FIT framework as that is path we are using going forward. Meaning CIT tests are now being merged into FIT.  Please contact @hohene if you have questions